src/components/events/events-card/events-card.jsx

```diff
@@ -76,8 +76,7 @@ const EventsCard = ({ event }) => {
     <div className="event-card-header">
       <EventStatus />
       <div className="event-card-name">
-        <h4>{series}</h4>
```

Comment:
Wondering if we should leave the event series in somewhere?

Reply:
A lot of the time we don't really know what event series an event falls into until the last minute. I'll put it back and it can just be optional for the event data to include it.

Reply:
so odd because they should be already defined in the master plan timeline, but if it's not our problem then yes +1 on making it optional :)